content/1.2/docs/serverless-workflows/troubleshooting.md

```diff
@@ -94,3 +94,27 @@ a CA which is not available to the workflow. The error in the workflow pod log u
 
 **Solution:**
 When deploying a workflow in a namespace other than the one where Sonataflow services are running (e.g., sonataflow-infra), there are essential steps to follow to enable persistence and connectivity for the workflow. See the following [steps](https://github.com/parodos-dev/orchestrator-helm-operator/blob/main/docs/release-1.2/README.md#additional-workflow-namespaces).
+
+### Problem: sonataflow-platform-data-index-service pods can't connect to the database on startup
+
+This can be cause by a network policy issue if you deployed RHDH Operator in a namespace different from `rhdh-operator`.
+
+**Cause:**
+Network policies may be restricting communication between pods within the `sonataflow-infra` namespace, preventing the SonataFlow platform pods from connecting to the database.
```

Comment:
> Network policies may be restricting communication

Why `may`? They are restricting communication for/to pods in the sonataflow-infra ns

I would put the `may` after:

> Network policies are restricting communication between pods within the `sonataflow-infra` namespace, they may be preventing the SonataFlow platform pods from connecting to the database.